scripts/nbconverted/apply_offset.py

```diff
@@ -0,0 +1,85 @@
+
+# coding: utf-8
+
+# In[1]:
+
+
+#-------------------------------------------------------------------------------------------------------------------------------
+# By Alexandra Lee (July 2018) 
+#
+# Apply offset vector
+#-------------------------------------------------------------------------------------------------------------------------------
```

Comment:
Maybe add some descriptions of how to use the script and what is the input/output of the script. Same comments to the other python scripts